src/deepsparse/v2/operators/engine_operator.py

```diff
@@ -29,12 +29,12 @@
 
 SUPPORTED_PIPELINE_ENGINES = [DEEPSPARSE_ENGINE, ORT_ENGINE]
 
-__all__ = ["EngineOperator"]
+__all__ = ["EngineOperator", "EngineOperatorInputs", "EngineOperatorOutputs"]
 
 
 class EngineOperatorInputs(BaseModel):
     engine_inputs: List = Field(description="engine_inputs")
-    engine: Optional[Engine] = Field(
+    engine: Optional[Any] = Field(
```

Comment:
Why this becomes `Any`?

Reply:
good catch. change I made while debugging. will switch back